service/core/client/src/main/scala/com/lightbend/lagom/internal/client/CircuitBreakersPanelInternal.scala

```diff
@@ -68,14 +80,19 @@ private[lagom] class CircuitBreakersPanelInternal(
         val callTimeout = breakerConfig.getDuration("call-timeout", MILLISECONDS).millis
         val resetTimeout = breakerConfig.getDuration("reset-timeout", MILLISECONDS).millis
 
+        import scala.collection.JavaConverters.asScalaBufferConverter
+        val exceptionWhitelist: Set[String] = breakerConfig.getStringList("exception-whitelist").asScala.toSet
```

Comment:
Also, with this _stringly-typed_ approach we don't break the API on `master` and we can probably backport to `1.3.x` too.